index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+<!-- Userform Title line  -->
+  <title>User Form</title>
+  <link rel="stylesheet" href="node_modules/bulma/css/bulma.css">
+  <link rel="stylesheet" href="css/mystyles.css">
+
+  <script defer src="https://use.fontawesome.com/releases/v5.14.0/js/all.js"></script>
+  <script src="javascript/script.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+<!-- Reference to user name and welcome page -->
```

Comment:
Good documentation/ commenting on what each piece of code means; really helped me to interpret what you did